modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -76,13 +90,13 @@ const createGhost = function (editor: Editor, elm, width, height) {
   return ghostElm;
 };
 
-const appendGhostToBody = function (ghostElm, bodyElm) {
+const appendGhostToBody = (ghostElm: HTMLElement, bodyElm: HTMLElement) => {
   if (ghostElm.parentNode !== bodyElm) {
     bodyElm.appendChild(ghostElm);
   }
 };
 
-const moveGhost = function (ghostElm, position, width, height, maxX, maxY) {
+const moveGhost = (ghostElm: HTMLElement, position, width: number, height: number, maxX: number, maxY: number) => {
```

Comment:
If you wanted to type `position`, it looks like it's type is `{ pageX: number; pageY: number }`